community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -504,7 +509,33 @@ public UniquenessConstraint uniquenessConstraintCreate( KernelStatement state, i
     }
 
     @Override
-    public Iterator<UniquenessConstraint> constraintsGetForLabelAndPropertyKey( KernelStatement state,
+    public MandatoryPropertyConstraint mandatoryPropertyConstraintCreate( KernelStatement state, int labelId,
```

Comment:
It feels like actual validation code should be in `ConstraintEnforcingEntityOperations` and `StateHandlingStatementOperations` should simply do `state.txState().constraintDoAdd( constraint )`. WDYT?